src/SilverpopConnector/SilverpopXmlConnector.php

```diff
@@ -3,9 +3,7 @@
 namespace SilverpopConnector;
 
 use Omnimail\Exception\Exception;
-use SilverpopConnector\SilverpopBaseConnector;
-use SilverpopConnector\SilverpopRestConnector;
-use SilverpopConnector\SilverpopConnectorException;
```

Comment:
Are these being removed because they're unused?
The commit message doesn't mention this part, and the PR doesn't have any description, so it's hard to tell if this is an intentional deletion.

Reply:
yeah - they are unused - my IDE highlighted them when I added the new use